dandi/metadata.py

```diff
@@ -270,9 +270,25 @@ def extract_wasDerivedFrom(metadata):
     models.Participant, "identifier", "subject_id", id=...
 )
 
-extract_wasGeneratedBy = extract_model_list(
-    models.Session, "name", "session_id", id=...
-)
+
+def extract_session(metadata):
```

Comment:
is that a
```suggestion
def extract_session(metadata: dict) -> list:
```
?
I should myself get into a habit to use type annotations